blackbox/docs/administration/ssl.txt

```diff
@@ -9,14 +9,17 @@ Secured Communications (SSL/TLS)
    *Secured Communications* is an Enterprise Edition feature.
 
 Secured communication allows you to encrypt traffic between the CrateDB node
-and a client. This applies to connections using HTTP (i.e. Admin UI, Crash, Rest)
-as well as connections using the PostgreSQL Wire Protocol (i.e. JDBC, psql).
+and a client. This applies to connections using HTTP (i.e. Admin UI, Crash, Rest),
```

Comment:
"REST"

link admin ui, link crash, link rest

(link them to the appropriate page in the guide)